api/MediaDevices.json

```diff
@@ -237,6 +237,185 @@
               "deprecated": false
             }
           }
+        },
+        "controller_option": {
+          "__compat": {
+            "description": "<code>controller</code> option",
+            "spec_url": "https://w3c.github.io/mediacapture-screen-share/#dom-displaymediastreamoptions-controller",
+            "support": {
+              "chrome": {
+                "version_added": "109"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "preferCurrentTab_option": {
+          "__compat": {
+            "description": "<code>preferCurrentTab</code> option",
```

Comment:
You may want to link to https://wicg.github.io/prefer-current-tab/ in a "spec_url" field

Reply:
I tried this, but then the linter failed because it reckons it's not an official spec...so I've just left this one.

Reply:
Shall we add this spec to https://github.com/w3c/browser-specs/blob/main/specs.json so that it's possible to link to it in a follow-up PR?